content/docs/command-reference/cache/dir.md

```diff
@@ -20,8 +20,8 @@ Helper to set the `cache.dir` configuration option. (See
 [cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory).)
 Unlike doing so with `dvc config cache`, this command transform paths (`value`)
 that are provided relative to the current working directory into paths
-**relative to the config file location**. They are required in the latter form
-for the config file.
+**relative to the config file location** of your own project. They are required
+in the latter form for the config file.
```

Comment:
Hmmm this is redundant. The config file is part of the project. The note in https://github.com/iterative/dvc.org/issues/986#issuecomment-586541617 suggests to clarify that absolute paths are preserved (with better language).